R/arg.R

```diff
@@ -87,6 +87,8 @@ arg_capture <- function(x) {
 
   arg_expr <- substitute_(x_expr, x_env)
   arg_env <- env_caller(2)
+
+  arg_expr <- .Call(interp_, arg_expr, arg_env, TRUE)
```

Comment:
I think this needs a line in the docs (i.e. this function will automatically interpolate)